NewSpring/ApollosProject
# [DEV-3748] Fixes Spacing on Prayer Tab

## DESCRIPTION

This PR corrects the height of the prayer tab for larger screens.

### What does this PR do, or why is it needed?

![Prayer-Space](https://user-images.githubusercontent.com/8878532/57631289-373adb80-756d-11e9-8d1d-1b16d4bfb59f.gif)

This PR add padding to the bottom of the `TabView` in the Prayer Tab so that it fills down to the top of the `TabBarNavigator` at the bottom of the view.

### How do I test this PR?

Fire up the sim on an iPhone X and go to the prayer tab to see there's no negative space.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] Closes [DEV-3748](https://newspring.atlassian.net/secure/RapidBoard.jspa?rapidView=71&modal=detail&selectedIssue=DEV-3748)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
